### ISSUE TITLE
implement back end for buying page

### ISSUE THREAD

**Issue by derrickluyen** [aloha-supply-co/ManoaCart#18]

**Comment by WenEddie** [comment]
Buying Page displays items being sold (information display: item name, image, price) from all users. When an item is selected to buy, a new page will only display the one item selected (additional information, ie, user who sold item, description).

**Comment by WenEddie** [comment]
Buying Page has been updated (By Jeffrey) to display all items that are currently being sold. 

When the "Buy" button is clicked, the website will redirect the user to a new page that displays more information based on the item selected. The "Notify Seller" button has not been implemented but it should email the seller.

I also added the Edit Item feature on the Selling page. When clicked, the website will redirect user to a new page and it will allow the seller to change/update the existing item's information. After the information has been submitted, the website will be redirected to the "Sell Your Item" page.

**Comment by WenEddie** [comment]
Buying Page has been updated (By Jeffrey) to display all items that are currently being sold. 

When the "Buy" button is clicked, the website will redirect the user to a new page that displays more information based on the item selected. The "Notify Seller" button has not been implemented but it should email the seller.

I also added the Edit Item feature on the Selling page. When clicked, the website will redirect user to a new page and it will allow the seller to change/update the existing item's information. After the information has been submitted, the website will be redirected to the "Sell Your Item" page.
